modules/network-observability-flow-filter-parameters.adoc

```diff
@@ -0,0 +1,58 @@
+:_mod-docs-content-type: REFERENCE
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+[id="network-observability-flowcollector-flowfilter-parameters_{context}"]
+= Flow filter configuration parameters
+The flow filter rules consist of mandatory and optional parameters.
+
+.Required configuration parameters
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`cidr`
+|  Provides the IP address and CIDR mask for the flow filter rule, supports IPv4 and IPv6 address format. If you want to match against any IP, you can use `0.0.0.0/0` or `::/0` for IPv4 and IPv6 respectively.
+
+|`action`
+| Describes the action to be taken for the flow filter rule. The possible values are `Accept` and `Reject`.
+
+* For the matching rule with the `Accept` action, this flow is allowed to be cached in the eBPF table, with the updated global metric `FlowFilterAcceptCounter`.
+* For the matching rule with Reject action this flow is dropped and not cached in eBPF table, with the updated global metric `FlowFilterRejectCounter`.
+* If the rule is not matched, the flow is allowed to be cached in eBPF table, with the updated global metric `FlowFilterNoMatchCounter`.
+|===
+
+
+.Optional configuration parameters
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`direction`
+| DestPorts defines the destination ports to filter flows by. To filter a single port, set a single port as an integer value, for example `destPorts: 80`. To filter a range of ports, use a "start-end" range, string format, for example `destPorts: "80-100"`.
+
+|`protocol`
+| Protocol of the flow filter rule. Possible values are `TCP`, `UDP`, `SCTP`, `ICMP`, `ICMPv6`.
+
+|`sourcePorts`
+| Defines the source port to filter flows by. To filter a single port, set a single port as an integer value, for example `sourcePorts: 80`. To filter a range of ports, use a "start-end" range, string format, for example `sourcePorts: "80-100"`.
+
+| `sourcePortRange`
+| Defines the source port range of the flow filter rule, using "80-100" format.
+
+| `dstPort`
+| DestPorts defines the destination ports to filter flows by. To filter a single port, set a single port as an integer value, for example `destPorts: 80`. To filter a range of ports, use a "start-end" range in string format, for example `destPorts: "80-100"`.
+
+| `dstPortRange`
+| Defines the single destination port of the flow filter rule.
+
```

Comment:
you are missing port and portRange when u use port it can be either srcport or dstport same applies for portRange either srcPortRange or dstPortRange

Reply:
since u are describing the APIs here then there is no range configs in the APIs
u have sourcePorts, dstPorts and ports and it can take single value or range using `start-end` format

Reply:
Okay I added [ports](https://github.com/openshift/openshift-docs/pull/74622/files#diff-c1f389cfa44881b2885e5709e478c820d4f98f140dbb208b231639b47ec4d159R41), so it I think this is complete now